user/module/probe_openssl_lib.go

```diff
@@ -104,7 +104,7 @@ func (m *MOpenSSLProbe) initOpensslOffset() {
 
 	// openssl 1.1.0a - 1.1.0l
 	for ch := 'a'; ch <= MaxSupportedOpenSSL110Version; ch++ {
-		m.sslVersionBpfMap["openssl 1.1.0"+string(ch)] = "openssl_1_1_1a_kern.o"
+		m.sslVersionBpfMap["openssl 1.1.0"+string(ch)] = "openssl_1_1_0a_kern.o"
```

Comment:
It looks like the wrong bytecode file was selected here?

Does fixing it here mean we don't need to fix `openssl_masterkey.h` anymore?

Reply:
> It looks like the wrong bytecode file was selected here?

Yes.

> Does fixing it here mean we don't need to fix `openssl_masterkey.h` anymore?

No.  That is not enough.

https://github.com/openssl/openssl/blob/OpenSSL_1_1_0a/ssl/ssl_locl.h#L1140-L1147
https://github.com/openssl/openssl/blob/OpenSSL_1_1_1a/ssl/ssl_locl.h#L1518-L1525

Note that in `ssl3_state_st` structure for OpenSSL versions 1.1.0a and 1.1.1a, the types of `read_mac_secret_size` and `write_mac_secret_size` members differ. On 64-bit systems, their sizes are not the same, resulting in a different offset for `client_random` member.